src/LSplots.jl

```diff
@@ -0,0 +1,85 @@
+using Pkg
+Pkg.instantiate()
+using TAMode
+using Plots
+using StaticArrays
+
+rr = TAMode.Lsparam(fill(0.2, 9));
+tps = @SVector Float64[60, 240];
+
+GasProp = @SVector[0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1];
+pYprop = Array{Float64}(undef, length(tps), length(GasProp));
+
+GasConc = @SVector Float64[64, 16, 4, 1, 0.25, 0];
+PROSConc = @SVector Float64[64, 16, 4, 1, 0.25, 0];
+pY = Array{Float64}(undef, length(tps), length(GasConc));
+
+GGdimers = vcat(zeros(9), ones(2), zeros(3), zeros(9), ones(2), zeros(5));
+PPdimers = vcat(zeros(11), ones(2), zeros(12), ones(2), zeros(3));
+GPdimers = vcat(zeros(13), 1, zeros(13), 1, zeros(2));
+GG = Array{Float64}(undef, length(tps), length(GasProp));
+PP = Array{Float64}(undef, length(tps), length(GasProp));
+GP = Array{Float64}(undef, length(tps), length(GasProp));
+
+
+#pY vs. Ligand Proportion
```

Comment:
Make each of these a function. Files within a package all get run on import, so these plots would be made when you load the package.